css21/visuren/position-fixed-001.xht

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html SYSTEM "about:legacy-compat">
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <title>CSS Test: position:fixed box's position calculated using absolute model</title>
+    <link rel="author" title="Chris Rebert" href="http://chrisrebert.com" />
+    <link rel="help" href="https://drafts.csswg.org/css2/visuren.html#choose-position" />
+    <link rel="help" href="https://drafts.csswg.org/css2/visudet.html#abs-non-replaced-width" />
+    <link rel="help" href="https://drafts.csswg.org/css2/visudet.html#abs-non-replaced-height" />
+    <link rel="match" href="position-fixed-001-ref.xht" />
+    <meta name="flags" content="" />
+    <meta name="assert" content="An element which is position:fixed should have its position calculated correctly using the absolute model when its top/bottom/left/right properties all use the default `auto` value and its parent is position:relative." />
```

Comment:
> Avoid use of expressions like "should work correctly", "should be 
implemented correctly", "should be supported correctly", "should be 
calculated correctly" and expressions like those in the meta text assert 
because they do not describe or explain in useful terms what the test is 
supposed to be testing, checking.